modules/ossm-control-plane-deploy.adoc

```diff
@@ -90,7 +90,7 @@ $ oc login https://{HOSTNAME}:6443
 $ oc new-project istio-system
 ----
 
-. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the full example found in "Customize the {ProductName} installation". You can customize the values as needed to match your use case. For production, you _must_ change the default Jaeger template.
+. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the example found in "Customize the {ProductName} installation". You can customize the values as needed to match your use case.  Note that for production use you _must_ change the default Jaeger template.
```

Comment:
s/Note that for production use you _must_ change the default Jaeger template./For production, you _must_ change the default Jaeger template./

I feel "Note that" is redundant here. I also think that "use" doesn't add any meaning to the sentence either.

Reply:
Updated text.